nodejs/src/lib/HivePublicKeyServiceClient.ts

```diff
@@ -165,11 +166,20 @@ export default class HivePublicKeyServiceClient {
      *
      * @param info The public key to store on the Hive Public Key Service.
      */
-    async create(info: PublicKeyStorePayload): Promise<void> {
+    async create(partnerId: string, keyId: string, exponent: string, modulus: string, expiration: string | number): Promise<void> {
+        let a: PublicKeyStorePayload
         try {
-            await this.client.post('/publickey', info);
+            const exp = typeof expiration === "string" ? checkExpiration(expiration) : expiration;
+            const payload: PublicKeyStorePayload = {
+                partnerId,
+                keyId,
+                exponent,
+                modulus,
+                expiration: exp
```

Comment:
This will return a number and the Type PublicKeyStorePayload has expiration as a number. So it will only accept number at the end of the day ( for other possible inputs) . Nice!